README.md

```diff
@@ -10,17 +10,36 @@ add support for different patterns of non-local control flow to WebAssembly.
 
 * See the [modified spec](https://webassembly.github.io/stack-switching/) for details.
 
-Original `README` from upstream repository follows…
+## Typed Continuations Proposal for WebAssembly
+
+This is a concrete proposal for realising the aims of the stack-switching proposal.
```

Comment:
This is a proposed solution addressing the aims of the stack-switching proposal.